src/core/components/settings/BlockSettings.tsx

```diff
@@ -18,7 +18,7 @@ export default function BlockSettings() {
   const updateBlockPropsRealtime = useUpdateBlocksPropsRealtime();
   const coreBlock = getBlockComponent(selectedBlock._type);
   const formData = { ...selectedBlock };
-  const dataBindingSupported = useBuilderProp("dataBindingSupport", false);
+  const dataBindingSupported = useBuilderProp("dataBindingSupport", true);
```

Comment:
the default value should be false. It needs to be enabled from props

Reply:
Ok. I'll check on that.